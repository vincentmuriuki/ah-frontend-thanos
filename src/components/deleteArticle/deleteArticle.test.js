import React from 'react';
import { shallow } from 'enzyme';
import DeleteArticle from './index';

describe('<DeleteArticle /> ', () => {
  const DeleteArticleComponent = shallow(
    <DeleteArticle
      onClick={jest.fn()}
    />,
  );
  test('renders the component', () => {
    expect(DeleteArticleComponent).toMatchSnapshot();
  });
});
