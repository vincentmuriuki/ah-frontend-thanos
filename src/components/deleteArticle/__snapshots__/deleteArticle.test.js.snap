// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<DeleteArticle />  renders the component 1`] = `
<div>
  <button
    className="btn btn-danger"
    onClick={[MockFunction]}
    type="submit"
  >
    DELETE ARTICLE
  </button>
</div>
`;
